avm/ptn/lz/sub-vending/modules/subResourceWrapper.bicep

```diff
@@ -1735,6 +1735,36 @@ module createAdditionalVnets 'br/public:avm/res/network/virtual-network:0.7.0' =
   }
 ]
 
+module createAdditionalLzVirtualWanConnection 'hubVirtualNetworkConnections.bicep' = [
```

Comment:
As this is added functionality should minor version be updated?

Reply:
so my thinking here not to bump it, is that it should of been in the original version, it was just missed 😢